core/src/main/java/com/amplifyframework/core/model/ModelSchema.java

```diff
@@ -119,8 +120,14 @@ public static ModelSchema fromModelClass(@NonNull Class<? extends Model> clazz)
             }
 
             for (Annotation annotation : clazz.getAnnotations()) {
-                ModelIndex modelIndex = createModelIndex(annotation);
-                if (modelIndex != null) {
+                if (annotation.annotationType().isAssignableFrom(Indexes.class)) {
```

Comment:
Previously, there was a bug here.  The logic was simply to iterate all of the `@Index` annotations of the class.  That works when only one `@Index` is defined.   However, when there is more than one `@Index`, it actually wraps _all_ the `@Index` annotations in a `@Indexes` annotation.  This was not accounted for.  

I've updated the logic to look for handle the single `@Index` scenario as well as the multiple `@Index` wrapped in `@Indeces` scenario.